### ISSUE TITLE
syntax error printed twice

### ISSUE THREAD

**Issue by sophiakoulen** [sophiakoulen/minishell#58]
```bash
|
```

Also, nul-character is printed => very bad.

**Comment by sophiakoulen** [comment]
removed the nul-character, but still printed twice

**Comment by sophiakoulen** [comment]
```bash
minishell$ |
minishell: syntax error near unexpected token `|'
minishell: syntax error near unexpected token `|'
minishell$ 
```

**Comment by sophiakoulen** [comment]
Fixed with 5d119e96eb7ab1b8d3e5ba47b55534932de15e00
